doc/installation-guide.rst

```diff
@@ -494,3 +540,167 @@ Known issues
+please rename the servar name or add puppet-master.novalocal to /etc/hosts associated to loaclhost.
+
+check first that the process puppet master is running (ps -ef | grep puppet) and finally run commands:
+
+.. code ::
+
+     service puppet start
+
+a certificate should have been created at /var/lib/puppet/ssl/certs/<puppet-agent-machine-name>.pem
+
+Check if the certificate has been registered by typing
+
+.. code ::
+
+     puppet cert list -all
+
+A certificate should be listed associated to the vm where the puppet agent has been installed (note: If there is no any certificate
+, please kill the puppet master process and start it again by typing puppet master. Type again 'puppet cert list -all' and check if 
+the corresponding certificate is listed)
+
+Now the following request can be performed in order to generate all files required to install software in the node where puppet agent
+has been installed:
+
+.. code ::
+
+     curl -v -k -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'X-Auth-Token: <token-id>' -H 'Tenant-Id: <tenant-id>' -X GET 'https://130.206.127.85:8443/puppetwrapper/v2/node/<hostname>/generate'
+
+the following response should have been obtained:
+
+.. code::
+
+     {"id":"<hostname>","groupName":"alberts","softwareList":[{"name":"testPuppet","version":"0.1","action":"INSTALL","attributes":[{"id":23119,"key":"clave","value":"valor"}]}],"manifestGenerated":true}
+
+where <hostname> is the hostname of the machine (with no damain). The corresponding manifests should have been created 
+together with the site.pp and the yaml file
+
+Finally for information purposes, we include the PuppetWrapper API for version2:
+
+.. code ::
+
+     POST /puppetwrapper/v2/node/{nodeName}/install 
+     ##json payload: 
+     {"attributes":[{"value":"valor","key":"clave","id":23119,"description":null}],"version":"0.1","group":"alberts","softwareName":"testPuppet"} 
+
+     POST /puppetwrapper/v2/node/{nodeName}/uninstall 
+     ##json payload: 
+     {"attributes":[{"value":"valor","key":"clave","id":23119,"description":null}],"version":"0.1","group":"alberts","softwareName":"testPuppet"} 
+
+     GET /puppetwrapper/v2/node/{nodeName}/generate ##will generate the following files in /etc/puppet/manifests 
+     
+add an import line to site.pp 
+generate the corresponding .pp file as group/nodeName.pp 
+
+.. code::
+
+     POST /puppetwrapper/module/{moduleName}/download 
+     ##payload : json as: {"url":�value�, �repoSource�:�value�} 
+
+Value on repoSource can be: git /svn 
+will download the source code from the given url under {moduleName} directory. 
+
+.. code::
+
+     DELETE /puppetwrapper/v2/node/{nodeName} 
+     ##will delete the node: nodeName 
+
+     DELETE /puppetwrapper/v2/module/{modulename} 
+     ##will delete the module: moduleName
+
+this requests will delete the node: nodeName and the module: moduleName.
```

Comment:
I still see the message at the end of this line. No newline at the end of this file.